components/tools/OmeroPy/src/omero/plugins/web.py

```diff
@@ -212,6 +214,28 @@ def config(self, args):
                     d["REWRITERULE"] = ""
                 d["NOW"] = str(datetime.now())
 
+            if server == "apache-fcgi":
+                c = file(self.ctx.dir / "etc" /
+                         "apache-mod_proxy_fcgi.conf.template").read()
+
+                if settings.APPLICATION_SERVER != settings.FASTCGITCP:
+                    self.ctx.die(
+                        679, "Apache mod_proxy_fcgi requires fastcgi-tcp")
+                fastcgi_external = '%s:%s' % (
+                    settings.APPLICATION_SERVER_HOST,
+                    settings.APPLICATION_SERVER_PORT)
+                d["FASTCGI_EXTERNAL"] = fastcgi_external
+
+                # Forcing a prefix for the entire server makes it easier to
+                # setup the proxy redirects
+                if d["FORCE_SCRIPT_NAME"] == '/':
+                    d["FORCE_SCRIPT_NAME"] = '/omero'
```

Comment:
You'd need separate `RewriteRule` and/or `ProxyPass` statements for each prefix (webclient, webgateway, webadmin, etc), otherwise requests for `/static` will be treated as a cgi request.